src/Components/article-content.jsx

```diff
@@ -0,0 +1,53 @@
+import { getArticleById } from "../../api"
+import { useEffect, useState } from "react"
+import { useNavigate } from 'react-router-dom'
+
+
+function ArticleContent(props){
+    
+    const articleId = props.articleId
+    const [article, setArticle] = useState(null)
+    const navigate = useNavigate()
+
+    useEffect(()=>{
+        getArticleById(articleId)
+        .then((response)=>{
+            console.log(response)
+            setArticle(response.article)
```

Comment:
This page would benefit a lot from some sort of loading state - if the request were to take a while, we don't want to have the user doubt that it might be broken instead of just loading